.buildbot.sh

```diff
@@ -89,7 +98,12 @@ cd $root
 
 RUSTDOCFLAGS="-Dwarnings" cargo doc --no-deps
 
-which cargo-deny | cargo install cargo-deny || true
-if [ "X`which cargo-deny`" != "X" ]; then
-    cargo-deny check license
-fi
+# Check licenses.
+wait "${cargo_deny_mdbook_pid}" || ( cat "${cargo_deny_mdbook_tmp}" && exit 1 )
+cargo-deny check license
+
+# Build the docs
+cd doc
+mdbook build
+test -d book
+cd ..
```

Comment:
Only a fairly minor nit, feel free to ignore if you prefer how it is.

I might suggest since the build root is in the `$root` variable already
`cd $root/doc` and `cd $root` instead of `cd doc` and `cd ..`

Reply:
Good idea! Fixed in the force push.